common/djangoapps/student/urls.py

```diff
@@ -38,27 +47,24 @@
         url(r'^auto_auth$', 'auto_auth'),
     )
 
+# add all student.views url patterns
```

Comment:
Reviewers: This comes afterward due to breaking up the above tuples for the conditional based on settings.